### DISCUSSION TITLE
How can I change the disc-color of a li

### DISCUSSION THREAD

**Discussion by georgfaust** [tailwindlabs/tailwindcss#11163] [Help] [answered]
I found this in a tutorial:

`prose prose-li:marker:text-red-500`

but it does not work, see: https://play.tailwindcss.com/RoJn8d4ync

**Comment by wongjn** [comment, marked as answer]
[Variant order matters in this case](https://tailwindcss.com/docs/hover-focus-and-other-states#ordering-stacked-modifiers), try `marker:prose-li:text-red-500`: https://play.tailwindcss.com/zXXuaOOans

**Reply by georgfaust** [reply]
nice thanks, seems like this was broken by the changes introduces in ordering a while ago...
